src/Servers/HttpSys/src/UrlPrefix.cs

```diff
@@ -15,8 +15,10 @@ private UrlPrefix(bool isHttps, string scheme, string host, string port, int por
             Scheme = scheme;
             Host = host;
             Port = port;
+            HostAndPort = string.Format(CultureInfo.InvariantCulture, "{0}:{1}", Host, Port);
```

Comment:
To confirm, UrlPrefixes are created once, not once per request?

Reply:
Right, these are server level config.